content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_SQL_Server_Deployment_Guide--Enabling_Always_Encrypted_with_YubiHSM_2/Basic_Setup_of_YubiHSM_2_and_SQL_Server.adoc

```diff
@@ -0,0 +1,48 @@
+== Basic Setup of YubiHSM 2 and SQL Server
+
+=== Installing and Configuring YubiHSM 2
+
+Install and configure the YubiHSM 2 device and software using the instructions in the following sections in the link:../YubiHSM_2_Windows_Deployment_Guide\--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server[YubiHSM 2 Windows Deployment Guide--Configure YubiHSM 2 Key Storage Provider for Microsoft Windows Server]:
+
+* link:../YubiHSM_2_Windows_Deployment_Guide\--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Install_the_YubiHSM_2_Tools_and_Software.adoc[Install the YubiHSM 2 Tools and Software]
```

Comment:
this link is broken, the document is named Install_the_YubiHSM_Tools_and_Software.adoc

Reply:
Rats! I really need to correct the filename rather than the link. But if I do that in this PR, it will break everywhere else. I will do a separate PR for that correction, containing filename plus all the links that target it. In the meantime, I will change the link so that it works.